File: PJDesignBackStageAPI/App.Model/Type2/GetType2ContentById.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace App.Model
{
    public class GetType2ContentByIdResponse : EditResponseBase
    {
        public int Id { get; set; }
        public int UnitId { get; set; }
        public string ThumbnailUrl { get; set; } = null!;
        public string ImageUrl { get; set; } = null!;
        public bool IsBefore { get; set; }
        public string Title { get; set; } = null!;
        public string Description { get; set; } = null!;
        public DateTime CreateDt { get; set; }
        public bool IsEnabled { get; set; }
        public bool IsFixed { get; set; }
        public string Content { get; set; } = null!;
        public List<Category>? Categories { get; set; }
    }
}

File: PJDesignBackStageClient/src/app/features/type2/feature-shared/models/get-type2-content-by-id.ts
import { EditResponseBase } from "src/app/shared/models/bases";
import { Category } from "src/app/shared/models/category";

export class GetType2ContentByIdResponse extends EditResponseBase {
  id: number;
  unitId: number;
  thumbnailUrl: string;
  imageUrl: string;
  isBefore: boolean;
  title: string;
  createDt: Date;
  isEnabled: boolean;
  isFixed: boolean;
  description: string;
  content: string;
  categories: Category[] | null;
}
